.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: java
 jdk:
-  - oraclejdk8
+  - openjdk11
```

Comment:
Can test on both?

Reply:
@pacomendes Would you please replace it with 

```yml
jdk:
  - openjdk8
  - openjdk11
```

Reply:
This will likely fail on openjdk8 as the -release option is not recognised.

Reply:
@pacomendes Raising a new PR to solve this problem https://github.com/apache/curator/pull/321